### ISSUE TITLE
Visualize .orbit output data in case of CLI version

### ISSUE THREAD

**Issue by alexkalmuk** [google/orbit#4817]
I'd like to know if there is a way to somehow get human readable data from `.orbit` (in my case it's `hello_2023_03_09_14_00_13.orbit`) output file. I'm using `WITH_GUI=OFF` version.

**Comment by alexkalmuk** [comment]
Thanks, I'll give a try to point 2 first.

**Comment by alexkalmuk** [comment]
This python stuff works, and it's possible to print out all the captured events. But now it seems that `OrbitClientGgp` doesn't work properly. I have a very simple program with 3 functions inside, which I'm profiling for 10 sec with GUI version (./bin/Orbit) and it shows me my 3 functions in the output with almost perfect estimation of time consumed per function. If I try CLI version:
```
./bin/OrbitClientGgp --file_directory hello_out --pid 80020
```
It find debug info successfully, profiles for 10 seconds, saves the trace and exits. When I'm loading this trace into Orbit GUI - it's shows nothing. When I'm analizing it with python scripts - no events mentioning any functions.

I plan to debug it somehow and would appreciate any ideas about what could be the reason.

**Comment by alexkalmuk** [comment]
> I plan to debug it somehow and would appreciate any ideas about what could be the reason.

Fixed by #4818

**Comment by florian-kuebler** [comment]
Thanks for the fix. Hope that it works now on your side!

**Comment by alexkalmuk** [comment]
Yes, it seems to be working. At least now I'm able to "Load capture" with a .orbit file generated by OrbitClientGgp, and it looks exactly like the capture produced by the gui version.
